tasks/prerequisites/setup-selinux.yml

```diff
@@ -21,6 +21,7 @@
   selinux:
     state: permissive
     policy: targeted
+  when: not (ansible_check_mode and nginx_selinux_enforcing)
```

Comment:
This should be `nginx_config_selinux_enforcing`

Reply:
Thanks for catching! Updates made.